zedc/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Recent Changes
+
+- **Enhancement:** Added `zedc status` command to display working tree status, diagnostics and system info in a developer's environment.
```

Comment:
I'm really liking the `zedc status` command
I may add some comments down below regarding what I would like to see in a future PR 🙏